tests/unit/test_dbapi_connection.py

```diff
@@ -219,8 +219,14 @@ def test_does_not_keep_cursor_instances_alive(self):
         # Connections should not hold strong references to the Cursor instances
         # they created, unnecessarily keeping them alive.
         gc.collect()
-        cursors = [obj for obj in gc.get_objects() if isinstance(obj, Cursor)]
-        self.assertEqual(len(cursors), 2)
+        cursor_count = 0
+        for obj in gc.get_objects():
+          try:
+            if isinstance(obj, Cursor):
+              cursor_count += 1
+          except ReferenceError:
```

Comment:
I think we might need some `# pragma: NO COVER` comments on lines/branches that aren't guaranteed to be hit each test run.

https://github.com/googleapis/python-bigquery/blob/82e0a7c231e823b025f7fa5b81c4695ced0f73ba/.coveragerc#L11

Reply:
done